### DISCUSSION TITLE
Using keyboard in android tablet

### DISCUSSION THREAD

**Discussion by JuanGandara** [exult/exult#552] [General]
Hi team,

First of all, thanks a lot. Ultima VII games are part of me. I have installed exult in an android tablet. It works fine but I am not able to input text whennit ask fir the Avatar name or to put a name to a save game. I can only use the quick sabe and this is too dangerous in this games xD.

Anyone have sufered the same problem?

PD: Bonus question, I have the Gog version of UVII. Anyone kniw how to obtain the Spanish versión or how to patch It for a Android tablet 

Thanks again!

**Comment by wench** [comment]
I doubt anyone has tested it with a tablet. I have only used it on a phone
and text input works correctly with the on screen keyboard for me on my
android phone.

Can't help with localised versions.

**Reply by JuanGandara** [reply]
Thanks a lot for your answer. Maybe IS a problem with my tablet. It allows me to enter the text, But when I press "enter" key It returns yo the Game field without any text.

Thanks a lot again!

**Reply by JuanGandara** [reply]
I am completelly stupid. There is a OK label under the field, if you press it the text is correctly stored.
